### PR TITLE
Combiner Script

### PR DESCRIPTION
**Pull request by babusid**
I'm not a huge fan of using branches to section off the different entries in the master document, so I did a little digging. Turns out, you can export singular pages to their own XML files in draw.io by going to File -> Export -> Export as XML (Making sure to uncheck "all pages"). This way, each page can be edited as a separate file. The script I've written in this PR allows you to put all of those "sub-diagrams" inside the diagrams folder, and run the script whenever you want to generate a full master diagram. I pulled some sheets from various branches that seemed up to date to provide a proof of concept. This PR description may seem scatterbrained (I wrote this and the script in like 5 minutes), so if you have questions, just lmk.

### CODE REVIEW

**Review by manthanand — DISMISSED** [review]

**Comment by Lakshay983** [comment]
I'm getting an issue where when i run the script it will combine the changes, but will generate a new copy of that sheet. 

Screenshot 2023-07-03 at 8 31 09 PM

Also, I renamed any mention of the .xml file extension to .drawio that way you can directly open those files in drawio by double-clicking in file explorer. The script worked the same when I did it. In the PR can you rename the .xml files to .drawio files? 

I can commit these changes if you want.

Rip csar 😭

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # WireWorld
-This repository contains information the wiring of all the electrical systems in the casr
+This repository contains information the wiring of all the electrical systems in the car.
+
+# Generator script
+
+To use the generator script, simply call `python diagram_generator.py`. This will combine the various xml files representing sub-diagrams into one large XML file which can be opened as a draw.io notebook.
```
Comment (manthanand) [inline README.md]: I think there should be more information about how it works. For example, what breaks it (people messing with folder structure, etc.).

**Review by Lakshay983 — DISMISSED** [review]

**Review by hannagnguyen — DISMISSED** [review]
good

**Review by Lakshay983 — APPROVED** [review]

**Review by Lakshay983 — APPROVED** [review]

**Review by Lakshay983 — DISMISSED** [review]

**Review by Lakshay983 — DISMISSED** [review]

**Review by Lakshay983 — APPROVED** [review]

**Review by IshDeshpa — APPROVED** [review]
LGTM!

**Pull request merged**